easybuild/easyconfigs/r/R/R-3.4.0-intel-2017a-X11-20170314.eb

```diff
@@ -104,7 +104,7 @@ exts_list = [
     ('markdown', '0.8', ext_options),
     ('mlbench', '2.1-1', ext_options),
     ('NLP', '0.1-10', ext_options),
-    ('mclust', '5.2.3', ext_options),
```

Comment:
@verdurin still available at https://cran.r-project.org/src/contrib/Archive/mclust/, which should be considered by EB (see definition of `ext_options`)?